README.md

```diff
@@ -1,14 +1,14 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Say Hello Microverse
+# `My portfolio mobile version`
```

Comment:
- You are missing to add linters into your pull request.

Reply:
Hello @gabrielcoder247 👋,

_This is an additional review as a result of the second opinion request._

Unfortunately, as at the last review, the linters for your PR are not active. Your last commit (https://github.com/gabrielcoder247/Gabs-portfolio/pull/4/commits/8dcbedcbc8e5c1a1654792451659b5d727951517) will show a green checkmark once it is active. Since this is your third review, you will have to repeat the week.
You can find out more [here](https://microverse.zendesk.com/hc/en-us/articles/1500004088561#).
Thank you.

Keep rocking 🚀!